humilis/utils.py

```diff
@@ -127,7 +127,8 @@ def load_section(self, section, files=None, params={}):
                 continue
 
             if len(this_data) != 1:
-                raise FileFormatError(filename, self.logger)
+                raise FileFormatError(filename, "Error loading layer section",
+                                      self.logger)
```

Comment:
There was a glitch on the way this exception was raised, adding a dummy message fix it.